extension/e2e-tests/sendPayment.test.ts

```diff
@@ -39,71 +39,38 @@ test("Send doesn't throw error when account is unfunded", async ({
   );
 });
 
-test("Send XLM payment to G address", async ({ page, extensionId }) => {
+test("Send XLM payments from multiple accounts to G Address", async ({
```

Comment:
this test now creates 2 accounts and sends payments with both to ensure that when we switch accounts, we properly retrieve the correct private key